Tests/ApolloCodegenTests/CodeGeneration/Templates/SelectionSet/SelectionSetTemplateTests.swift

```diff
@@ -456,4 +506,181 @@ class SelectionSetTemplateTests: XCTestCase {
     expect(actual).to(equalLineByLine(expected, atLine: 6, ignoringExtraLines: true))
   }
 
+
+  // MARK: - Field Accessors - Scalar
```

Comment:
Should there be a test to ensure field accessors are "lower first" cased?

Reply:
Will do in next PR. Thanks!